src/pages/Beta/Stake/DetailModal/EarnedWidget/index.tsx

```diff
@@ -61,7 +61,7 @@ const EarnedWidget: React.FC<Props> = ({ stakingInfo }) => {
         <Box>
           <Stat
             title={t('dashboardPage.earned_weeklyIncome')}
-            stat={numeral(weeklyRewardUSD).format('$0.00a')}
+            stat={numeral(weeklyRewardUSD && isNaN(weeklyRewardUSD) ? weeklyRewardUSD : 0).format('$0.00a')}
```

Comment:
this is not correct. condition is logically reversed. also condition should come early not inside numeral. 
it should be something like this
```js
!isNaN(weeklyRewardUSD) ? numeral(weeklyRewardUSD).format('$0.00a') : '0'
```

Also do this change for all places where you used isNaN